packages/website/src/components/editor/useSandboxServices.ts

```diff
@@ -34,6 +35,7 @@ export const useSandboxServices = (
 ): Error | SandboxServices | undefined => {
   const [services, setServices] = useState<Error | SandboxServices>();
   const [loadedTs, setLoadedTs] = useState<string>(props.ts);
+  const { isDarkTheme } = useColorMode();
```

Comment:
`isDarkTheme` is deprecated—you should see a warning in the console in development (it probably is not transparent enough, considering most people don't develop with their consoles open all the time). You should use [`const { colorMode } = useColorMode()`](https://docusaurus.io/docs/api/themes/configuration#use-color-mode) instead